src/hearings/containers/request-hearing/hearing-venue/hearing-venue.component.ts

```diff
@@ -1,42 +1,42 @@
-import {Component, OnDestroy, OnInit, ViewChild} from '@angular/core';
-import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { SearchLocationComponent } from '@hmcts/rpx-xui-common-lib';
-import {LocationByEPIMSModel} from '@hmcts/rpx-xui-common-lib/lib/models/location.model';
+import { LocationByEPIMSModel } from '@hmcts/rpx-xui-common-lib/lib/models/location.model';
 import { select, Store } from '@ngrx/store';
-import {Observable, of, Subscription} from 'rxjs';
-import {map} from 'rxjs/operators';
+import { Observable, of, Subscription } from 'rxjs';
+import { map } from 'rxjs/operators';
 import * as fromHearingStore from '../../../../hearings/store';
-import {HearingLocationModel} from '../../../models/hearingLocation.model';
-import {ACTION} from '../../../models/hearings.enum';
-import {HearingsService} from '../../../services/hearings.service';
-import {RequestHearingPageFlow} from '../request-hearing.page.flow';
+import { HearingLocationModel } from '../../../models/hearingLocation.model';
+import { ACTION } from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
 
 @Component({
   selector: 'exui-hearing-venue',
   templateUrl: './hearing-venue.component.html',
   styleUrls: ['./hearing-venue.component.scss']
 })
-export class HearingVenueComponent extends RequestHearingPageFlow  implements OnInit, OnDestroy {
+export class HearingVenueComponent extends RequestHearingPageFlow implements OnInit, OnDestroy {
   public locationType: string;
-  public selectedLocations$: Observable<LocationByEPIMSModel[]>;
-  public locationsFound$: Observable<LocationByEPIMSModel[]>;
+  public locationsDisplayedInDrop: LocationByEPIMSModel[];
```

Comment:
tricky name locationsDisplayedInDrop, can it just be locationDisplayed or displayedLocation?